src/nvim/main.c

```diff
@@ -2188,38 +2189,39 @@ static void usage(void)
 {
   signal_stop();              // kill us with CTRL-C here, if you like
 
-  os_msg(_("Usage:\n"));
-  os_msg(_("  nvim [options] [file ...]\n"));
-  os_msg(_("\nOptions:\n"));
-  os_msg(_("  --cmd <cmd>           Execute <cmd> before any config\n"));
-  os_msg(_("  +<cmd>, -c <cmd>      Execute <cmd> after config and first file\n"));
-  os_msg(_("  -l <script> [args...] Execute Lua <script> (with optional args)\n"));
-  os_msg(_("  -S <session>          Source <session> after loading the first file\n"));
-  os_msg(_("  -s <scriptin>         Read Normal mode commands from <scriptin>\n"));
-  os_msg(_("  -u <config>           Use this config file\n"));
-  os_msg("\n");
-  os_msg(_("  -d                    Diff mode\n"));
-  os_msg(_("  -es, -Es              Silent (batch) mode\n"));
-  os_msg(_("  -h, --help            Print this help message\n"));
-  os_msg(_("  -i <shada>            Use this shada file\n"));
-  os_msg(_("  -n                    No swap file, use memory only\n"));
-  os_msg(_("  -o[N]                 Open N windows (default: one per file)\n"));
-  os_msg(_("  -O[N]                 Open N vertical windows (default: one per file)\n"));
-  os_msg(_("  -p[N]                 Open N tab pages (default: one per file)\n"));
-  os_msg(_("  -R                    Read-only (view) mode\n"));
-  os_msg(_("  -v, --version         Print version information\n"));
-  os_msg(_("  -V[N][file]           Verbose [level][file]\n"));
-  os_msg("\n");
-  os_msg(_("  --                    Only file names after this\n"));
-  os_msg(_("  --api-info            Write msgpack-encoded API metadata to stdout\n"));
-  os_msg(_("  --clean               \"Factory defaults\" (skip user config and plugins, shada)\n"));
-  os_msg(_("  --embed               Use stdin/stdout as a msgpack-rpc channel\n"));
-  os_msg(_("  --headless            Don't start a user interface\n"));
-  os_msg(_("  --listen <address>    Serve RPC API from this address\n"));
-  os_msg(_("  --remote[-subcommand] Execute commands remotely on a server\n"));
-  os_msg(_("  --server <address>    Specify RPC server to send commands to\n"));
-  os_msg(_("  --startuptime <file>  Write startup timing messages to <file>\n"));
-  os_msg(_("\nSee \":help startup-options\" for all options.\n"));
+  printf("%s", _("Usage:\n"));
```

Comment:
also in cases like this. gettext formatstring is used anyway in a lot of other places.